### PR TITLE
Fix for small screens with large fonts don't fit all content

### LINKED ISSUE #2931: Quick Order list: small screens with large fonts dont fit all content
Video: https://screenshot.click/31-13-nhlcn-llnnz.mp4

### PR DESCRIPTION
**Pull request by eugenekasimov**
### PR Summary: 

This PR fixes an issue when the total price of an item on mobile goes out of screen. This happens when an item has `compare_at_price` and large fonts, so in that case there is just not enough room to fit the total price on a small screen. 

### Why are these changes introduced?

Fixes #2931 .

### What approach did you take?

I changed the way we display ~~the old price~~ and the current one. They used to be in one line. Now I place them in two different lines under each other. 

### Visual impact on existing themes

The layout of  ~~the old price~~ and the current one has been changed for mobile. 

<details><summary>Before</summary>
258262137-6e672390-fbc3-4dd7-92c4-1c4cf50082cb
</details>

<details><summary>After</summary>
258263003-c27329aa-4dc3-42ef-8ce9-4ffc657dc605
</details>

### Testing steps/scenarios

- [ ] Add the quick order list section to the product page.
- [ ] Go to a product that has compare_at_price (i.e Louise Slide Sandal)
- [ ] In the Quick Order List settings choose Show images.
- [ ] In Global Theme settings set fonts to max
- [ ] in the editor or using the dev tool go to the mobile view.
- [ ] In the quick order list increase the qty of an item that has compare_at_pricer to make the total price be a four-digit number and make sure it fits the screen. 

### Demo links

- [Editor](https://admin.shopify.com/store/os2-demo/themes/140437225494/editor)

### Checklist
- [ ] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [ ] Requested review from UX (Only for changes that are affecting the experience or perceivable visual details)
- [ ] Created a ticket for the [help.shopify.com](https://help.shopify.com) documentation team about updates to theme settings. (Internal-only task)
- [ ] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [ ] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [ ] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [ ] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [ ] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

File: assets/quick-order-list.css [hunk 1]
```diff
@@ -187,6 +187,12 @@ quick-order-list .quantity__button {
   align-items: center;
 }
 
+@media screen and (max-width: 749px) {
+  .variant-item__discounted-prices {
+    display: block;
```
[the remark this answers is not in the document]
Comment (eugenekasimov) [reply assets/quick-order-list.css:192]: Hey Andrew, thanks for a good idea. I like this approach.

**Review by sofiamatulis — APPROVED** [review]
Looks good 👌

File: assets/quick-order-list.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (edmund-teh) [reply assets/quick-order-list.css]: Sick! That's a great call, @andrewetchen... sorry, late to catching up with this one. Thank you! 🚀

**Review by edmund-teh — APPROVED** [review]
Tested on my phone, iPad, and on Desktop. All looks great and works as advertised. Thanks @eugenekasimov!

**Pull request merged**